src/ExpressionEvaluator/CSharp/Test/ResultProvider/ValueFormattingTests.cs

```diff
@@ -214,8 +214,8 @@ public void String()
             Assert.Equal("\"\\uffff\"", FormatValue(s, useHexadecimal: true));
 
             string multiByte = "\ud83c\udfc8";
-            Assert.Equal(string.Format(format, "🏈"), FormatValue(multiByte));
-            Assert.Equal(string.Format(format, "🏈"), FormatValue(multiByte, useHexadecimal: true));
+            Assert.Equal(string.Format(format, "\\ud83c\\udfc8"), FormatValue(multiByte));
+            Assert.Equal(string.Format(format, "\\ud83c\\udfc8"), FormatValue(multiByte, useHexadecimal: true));
```

Comment:
> But we do not have platform APIs that help us determine the Unicode category of Unicode code points that are represented by surrogate pairs.

@gafter, I'm not sure what you mean by this statement, `char.GetUnicodeCategory(string, int)` is designed specifically for this purpose.

For example, the sumerian character `𒅴´ is a surrogate pair. Because it is comprised of two separate Unicode characters, it cannot be represented as a character literal (`char c = '𒅴'` is invalid). Because of this, you cannot use `char.GetUnicodeCategory('𒅴')`. However, it is valid in a string (string s = "𒅴") and can therefore be passed to the other overload: `char.GetUnicodeCategory("𒅴", 0)` returns `OtherLetter` (`char.GetUnicodeCategory("𒅴", 1)` returns `Surrogate`).